test/basic.js

```diff
@@ -53,7 +53,7 @@ t.test('spawn execPath', function (t) {
 })
 
 t.test('exec shebang', function (t) {
-  var child = cp.exec(fixture + ' xyz')
+  var child = spawn(fixture, ['xyz'])
```

Comment:
See https://github.com/bcoe/signal-exit/pull/15.  It works fine, but exec() uses a real `/bin/sh` on Linux, so a bunch of the tests gets weird results.